package-lock.json

```diff
@@ -0,0 +1,6258 @@
+{
```

Comment:
could you remove this file? I prefer to use yarn in this project

Reply:
This one is an accident. I'll remove it when I'm home. (Can I put it in `.gitignore`?)

Reply:
> Can I put it in .gitignore?

Sure